Gems/PhysX/Code/Tests/PhysXColliderComponentModeTests.cpp

```diff
@@ -24,7 +24,7 @@
 namespace UnitTest
 {
     class PhysXColliderComponentModeTest
-        : public ToolsApplicationFixture
+        : public ToolsApplicationFixture<false>
```

Comment:
So we don't check for leaks in the PhysXColliderCompoentModeTest fixture?

Reply:
Well we could move the Per-Module Generic Class Info to the stateless allocator.
It is not in a critical path for runtime use and it is used to store instances to template classes reflected with the current module memory space.

Alternatively could we move the Per-Module Generic Class info to be an AZ::EnvironmentVariable.
Do those get checked in the memory leak checks?